components/github/sources/webhook-event/webhook-event.mjs

```diff
@@ -7,7 +7,7 @@ export default {
   name: "New Webhook Event (Instant)",
```

Comment:
Here you need to change the `key` property from `key: "github-webhook-events",` to `key: "github-webhook-event",` that way we can get rid of the github actions error is there

Reply:
I prefer to rename folder name.

[Context here](https://pipedream-users.slack.com/archives/G01J50YGJGP/p1665666317535359?thread_ts=1665663517.003379&cid=G01J50YGJGP)